src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -124,12 +124,12 @@ static PcsStartup()
     /// </summary>
     /// <param name="builder"></param>
     /// <param name="addKeyVault">Use KeyVault for secrets?</param>
-    /// <param name="addRedis">Use Redis for caching?</param>
+    /// <param name="addRedisAuth">Use authenticated connection for Redis?</param>
     /// <param name="addSwagger">Add Swagger UI?</param>
     internal static async Task ConfigurePcs(
         this WebApplicationBuilder builder,
         bool addKeyVault,
-        bool addRedis,
+        bool addRedisAuth,
```

Comment:
`authRedis` might be better - this sounds like we're addng some auth based on / stored in Redis

Reply:
Good point, thanks. Will change